docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,732 @@
+
+3. Insert this code into the first line after the `<dict>`:
+
+   ```{class="line-numbers linkable-line-numbers"}
+     <key>NSLocalNetworkUsageDescription</key>
+     <string>Smart Machine App requires access to your device's local network to connect to your devices.</string>
+     <key>NSBonjourServices</key>
+     <array>
+         <string>_rpc._tcp</string>
+     </array>
+   ```
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist-insert.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The info.plist file open in VS Code. An arrow labeled 'insert here' indicates that you should insert the code right after the fourth line in the file.">}}
+
+   It should look like this:
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist-pasted.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The resulting file. After the non-indented <dict> tag on the fourth line, the pasted key, string, key, and array (containing an indented string) are indented inside the dict tag, spanning lines five through ten. The pasted content is at the same level of indentation as the rest of the keys and strings below it.">}}
+
+### Create the main file
+
+1. Open the <file>main.dart</file> file under the <file>lib/</file> directory.
+
+2. Replace the contents of this file with the following code to make up the login screen for your app:
+
+   ```{class="line-numbers linkable-line-numbers"}
+   import 'package:flutter/material.dart';
+   import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+   void main() async {
+      await dotenv.load();
+     runApp(MyApp());
+   }
+
+   class MyApp extends StatelessWidget {
+     const MyApp({super.key});
+
+     @override
+     Widget build(BuildContext context) {
+       return MaterialApp(
+         title: 'Smart Machine App',
+         theme: ThemeData(
+           colorScheme: ColorScheme.fromSeed(seedColor: Colors.purple),
+         ),
+         home: MyHomePage(),
+       );
+     }
+   }
+
+   class MyHomePage extends StatelessWidget {
+     @override
+     Widget build(BuildContext context) {
+       return Scaffold(
+         body: Center(
+           child: Column(
+             mainAxisAlignment: MainAxisAlignment.center,
+             children: [
+               Text('Smart Machine App'),
+               SizedBox(height: 16),
+               ElevatedButton(onPressed: null, child: Text('Login')),
+             ],
+           ),
+         ),
+       );
+     }
+   }
+
+
+   ```
+
+   If you chose a name other than `Smart Machine App`, edit lines 13 and 30 with your own app title.
+
+### Launch the app
+
+In this section of the tutorial we'll build the app and see what it looks like so far.
```

Comment:
Nit: talking to the user:

```suggestion
You now have enough of your new app configured to be able to build it.
Follow the steps below to build and preview the current state of your app.
```

or something similar. Vflexible.

Reply:
```suggestion
You now have enough of your new app coded to be able to build and test a rendering of it.

Follow the steps below to build and preview the current state of your app.
```
I would prefer to use a word other than "configured" to save that for JSON files, and I think some folks might not know what "build" means in this context. Does this suggestion still look technically accurate?